src/Symfony/Component/Translation/Bridge/Crowdin/CrowdinProvider.php

```diff
@@ -128,39 +124,37 @@ public function read(array $domains, array $locales): TranslatorBag
                     $response = $this->downloadSourceFile($fileId);
                 }
 
-                $responses[] = [
-                    'response' => $response,
-                    'locale' => $locale,
-                    'domain' => $domain,
-                ];
+                $responses[] = [$response, $locale, $domain];
             }
         }
 
-        foreach ($responses as $responseData) {
-            /** @var ResponseInterface $response */
-            $response = $responseData['response'];
-
+        /* @var ResponseInterface $response */
+        $downloads = [];
+        foreach ($responses as [$response, $locale, $domain]) {
             if (204 === $response->getStatusCode()) {
-                $this->logger->error(sprintf('No content in exported file. Message: "%s".', $response->getContent(false)));
+                $this->logger->error(sprintf('No content in exported file: "%s".', $response->getContent(false)));
 
                 continue;
             }
 
             if (200 !== $response->getStatusCode()) {
-                $this->logger->error(sprintf('Unable to export file. Message: "%s".', $response->getContent(false)));
+                $this->logger->error(sprintf('Unable to export file: "%s".', $response->getContent(false)));
 
                 continue;
             }
 
-            $exportResponse = $this->filesDownloader->request('GET', $response->toArray()['data']['url']);
```

Comment:
Separate HTTP Client `filesDownloader` was passed to the Crowdin Provider as we can't use the default pre-configured `client` as a download URL is not a Crowdin link, this is S3 link. 
Downloading a file using the `$this->client` failing because of pre-configured base URL and Authorization header

Reply:
See https://github.com/symfony/symfony/pull/41146#discussion_r628916683